src/_scss/custom.scss

```diff
@@ -6,7 +6,9 @@
 
 // ***** PAGES
 @import './pages/homepage';
-@import './pages/searchpage';
+@import './pages/search/searchPage';
+@import './pages/search/searchSidebar';
```

Comment:
Let's move the import statements for search page components inside `searchPage.scss` so that those styles are constrained to just the search page, rather than being universal across the site. Universal styles were what caused the Sass to become unmaintainable last time.